hw/bsp/dwm1001-dev/pkg.yml

```diff
@@ -41,3 +41,9 @@ pkg.deps.SOFT_PWM:
 
 pkg.deps.UARTBB_0:
     - "@apache-mynewt-core/hw/drivers/uart/uart_bitbang"
+
+pkg.deps.LIS2DH12_ONB:
+    - "@apache-mynewt-core/hw/drivers/sensors/lis2dh12"
+    
+pkg.init:
```

Comment:
if it was changed to:
```yml
pkg.init.LIS2DH12_ONB:
```
whole config_lis2dh12_sensor()  function could be ifdef'ed-out not just its body

Reply:
Fist line of commit message would also be more consistent with what is used in mynewt if it looked like text that was chosen for PR title:
```
hw/bsp/dwm1001-dev: Add LIS2DH12 accelerometer support
```
instead of "Add LIS2DH12 accelerometer support for DWM1001-dev boards"